src/FSharp.Build/WriteCodeFragment.fs

```diff
@@ -23,6 +23,9 @@ type WriteCodeFragment() as this =
                 raise TaskFailed)
             fmt
 
+    [<Struct>]
+    type EscapedValue = { Escaped: string; Raw: string }
```

Comment:
You cannot declare a type within another type.
Move this declaration above the `WriteCodeFragment` declaration at top of this file.